packages/flutter/lib/src/cupertino/localizations.dart

```diff
@@ -79,6 +79,17 @@ abstract class CupertinoLocalizations {
   // The global version uses date symbols data from the intl package.
   String datePickerMonth(int monthIndex);
 
+  /// Month that is shown in [CupertinoDatePicker] spinner corresponding to
+  /// the given month index in [CupertinoDatePickerMode.monthYear] mode.
+  ///
+  /// Examples: datePickerMonth(1) in:
+  ///
+  ///  - US English: January
+  ///  - Korean: 1월
+  ///  - Russian: Январь
```

Comment:
Can you add a Russian example to the datePickerMonth docs above this?  That way it might help someone see that there is a difference between Январь and января.